src/System.Xml.XmlDocument/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
I understand.  But it got updated somehow, and I believe our policy is still to have these checked in as locked: true.  @weshaggard can clarify.